__tests__/app.test.js

```diff
@@ -103,4 +106,39 @@ describe("GET /api/topics", () => {
     });
     
     
-  })  
\ No newline at end of file
+  })  
+
+  describe("when given a valid article ID returns with comments for correct artice", () => {
+    it("returns an array of comments objects with correct properties", () => {
+      return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({body}) => {
+        expect(body).toBeInstanceOf(Array);
+        expect(body).toBeSorted({ descending: true });
+        expect(body.length).toBeGreaterThan(0);
+        expect(typeof body[0]).toBe("object");
+        expect(body[0]).toHaveProperty("comment_id");
+        expect(body[0]).toHaveProperty("body");
+        expect(body[0]).toHaveProperty("votes");
+        expect(body[0]).toHaveProperty("author");
+        expect(body[0]).toHaveProperty("created_at");
+      });
+    });
+    it("404: responds with a bad request, no article exists", () => {
+      return request(app)
+        .get("/api/articles/205345345/comments")
+        .expect(404)
+        .then(({ body }) => {
+          expect(body.msg).toBe("404: Article not found");
+        });
+    });
+    it("400: responds with an Internal server error", () => {
+      return request(app)
+        .get("/api/articles/sdfsdfsdf/comments")
+        .expect(400)
+        .then(({ body }) => {
+          expect(body.msg).toBe("400: Invalid article_id");
+        });
+    });
+  })
```

Comment:
Nice stuff so far, need to check that if you made a request for a valid existing article that has no comments, it would return with a 200 and an empty array of comments, rather than an error